llvm/lib/Target/LoongArch/AsmParser/LoongArchAsmParser.cpp

```diff
@@ -1176,7 +1176,7 @@ void LoongArchAsmParser::emitLoadAddressTLSIE(MCInst &Inst, SMLoc IDLoc,
   Insts.push_back(LoongArchAsmParser::Inst(
       LD, LoongArchMCExpr::VK_LoongArch_TLS_IE_PC_LO12));
 
-  emitLAInstSeq(DestReg, DestReg, Symbol, Insts, IDLoc, Out);
+  emitLAInstSeq(DestReg, DestReg, Symbol, Insts, IDLoc, Out, true);
```

Comment:
I like adding comments to parameters without variable names, like this: `/*RelaxHint=*/true`.

Reply:
Thanks. I will modified this.